pigeon/comms/services/common.py

```diff
@@ -43,12 +43,14 @@ class HearbeatService(MavlinkService):
     def __init__(self,
                  commands: queue.Queue,
                  disconnect: Callable,
+                 timeout: int,
```

Comment:
Let's add a default value to this, maybe 15?

Otherwise we will get a crash when starting pigeon as we do not pass a timeout in `pigeon/comms/uav.py`:

```
Traceback (most recent call last):
  File "/.../python3.10/threading.py", line 1016, in _bootstrap_inner
    self.run()
  File "/.../python3.10/threading.py", line 953, in run
    self._target(*self._args, **self._kwargs)
  File "pigeon/pigeon/comms/uav.py", line 85, in <lambda>
    self.thread = Thread(target=lambda: self._runEventLoop(), args=[])
  File "pigeon/comms/uav.py", line 200, in _runEventLoop
    HearbeatService(self.commands, self.disconnect),
TypeError: HearbeatService.__init__() missing 1 required positional argument: 'timeout'
```

Reply:
I've added a default value of 15.